consensus/core/src/metrics.rs

```diff
@@ -476,6 +491,12 @@ impl NodeMetrics {
                 "Index of the last commit.",
                 registry,
             ).unwrap(),
+            last_commit_time_diff: register_histogram_with_registry!(
+                "last_commit_time_diff",
+                "The time diff between the last commit and previous one.",
```

Comment:
its the time diff between the current commit and last commit right?

Reply:
Yep that's it